### PR TITLE
fix build bug

### LINKED ISSUE #12: <url> Some(http://github.com/scalaz/effect) </url>
```
sbt makePom
cat target/scala-2.12/scalaz-ioeffect_2.12-0.0.1-SNAPSHOT.pom
```

```xml
<?xml version='1.0' encoding='UTF-8'?>
<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
    <modelVersion>4.0.0</modelVersion>
    <groupId>org.scalaz</groupId>
    <artifactId>scalaz-ioeffect_2.12</artifactId>
    <packaging>jar</packaging>
    <description>scalaz-ioeffect</description>
    <url>http://github.com/scalaz/effect</url>
    <version>0.0.1-SNAPSHOT</version>
    <licenses>
        <license>
            <name>BSD3</name>
            <url>https://opensource.org/licenses/BSD-3-Clause</url>
            <distribution>repo</distribution>
        </license>
    </licenses>
    <name>scalaz-ioeffect</name>
    <inceptionYear>2017</inceptionYear>
    <organization>
        <name>org.scalaz</name>
        <url>http://github.com/scalaz/effect</url>
    </organization>
    <scm>
        <url> Some(http://github.com/scalaz/effect) </url>
```

`addSbtPlugin("com.fommil"    % "sbt-sensible" % "2.4.1")` bug ? 🤔
https://gitlab.com/fommil/sbt-sensible/blob/36ed88c76e7b249571b951c8ac653ba66b5b00e7/src/main/scala/SonatypePlugin.scala#L131
 /cc @fommil

### PR DESCRIPTION
**Pull request by fommil**
close #12

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -65,19 +64,14 @@ object ProjectPlugin extends AutoPlugin {
...
         "-unchecked",
         "-explaintypes",
-        "-Ywarn-value-discard",
-        "-Ywarn-numeric-widen",
-        "-Ywarn-dead-code",
         "-Ypartial-unification",
         "-Xlog-free-terms",
         "-Xlog-free-types",
-        "-Xlog-reflective-calls",
-        "-Yrangepos",
```
Comment (hrhino) [inline project/ProjectPlugin.scala:78]: `rangepos` broken?

File: project/ProjectPlugin.scala [hunk 2]
```diff
@@ -65,19 +64,14 @@ object ProjectPlugin extends AutoPlugin {
       MonadicFor,
       Silencer,
       scalacOptions ++= Seq(
-        "-language:_",
         "-unchecked",
         "-explaintypes",
-        "-Ywarn-value-discard",
-        "-Ywarn-numeric-widen",
-        "-Ywarn-dead-code",
```
Comment (hrhino) [inline project/ProjectPlugin.scala:73]: this can come back with fommil#1

File: project/ProjectPlugin.scala [hunk 3]
```diff
@@ -65,19 +64,14 @@ object ProjectPlugin extends AutoPlugin {
       MonadicFor,
       Silencer,
       scalacOptions ++= Seq(
-        "-language:_",
         "-unchecked",
         "-explaintypes",
-        "-Ywarn-value-discard",
```
Comment (hrhino) [inline project/ProjectPlugin.scala:71]: I like this one too...

**Review by hrhino — CHANGES_REQUESTED** [review]

File: project/ProjectPlugin.scala (hunk 3 above)
Comment (fommil) [reply project/ProjectPlugin.scala:71]: these are already there, we have `sbt-sensible` :wink:

Comment (hrhino) [reply project/ProjectPlugin.scala:71]: 💯

**Pull request merged**